testsuite/HOWTO.md

```diff
@@ -0,0 +1,176 @@
+The unit tests and the test data are bundled together in the package **MDAnalyisTests**. In order to run the tests, this package must be installed in addition to MDAnalysis.
+
+The tests also rely on the `pytest` and `numpy` packages, and require both to run.
+
+# Users
+Install [MDAnalysisTests](MDAnalysisTests) via
+```
+pip install --upgrade MDAnalysisTests
+```
+or download the tar file, unpack, and run `python setup.py install`.
+
+Run the tests by invoking
+```
+python -c 'from MDAnalysisTests import run; run()'
+```
+
+All tests should pass (i.e. no **FAIL**, **ERROR**); *SKIPPED* or *XFAIL* are ok. For anything that fails or gives an error [ask on the user mailing list](http://users.mdanalysis.org) or [raise an issue](/MDAnalysis/mdanalysis/issues).
+
+# Developers #
+
+All tests should pass (i.e. no **FAIL**, **ERROR**); *SKIPPED* or *XFAIL* are ok. For anything that fails or gives an error **fix your code** (or *raise an issue*).
+
+## Recommended ##
+Use the tests from the [git source repository](Source), which are located in the [testsuite/MDAnalysisTests](https://github.com/MDAnalysis/mdanalysis/tree/develop/testsuite) directory:
+```
+cd testsuite/MDAnalysisTests
+pytest  --disable-pytest-warnings
+```
+
+Running the tests serially can take some time (>20min) depending on the performance of your computer. You can also run the tests in parallel. To do so you will need `pytest-xdist` installed
+```
+pip install pytest-xdist
+pytest  --disable-pytest-warnings --numprocesses 4
+```
+
+(Try increasing the number of processes; with 24 processes on 12 cores (+hyperthreading) this took ~40 seconds; in serial it takes ~30 min).
```

Comment:
please remove the times and only mention that xdist can improve performance. Even travis machines now need less then 30 minutes.